Typescript type of Uppy.close is incorrect

### Initial checklist

- [X] I understand this is a bug report and questions should be posted in the [Community Forum](https://community.transloadit.com/)
- [X] I searched [issues](https://github.com/transloadit/uppy/issues?q=is%3Aissue) and couldn’t find anything (or linked relevant results below)

### Link to runnable example

_No response_

### Steps to reproduce

In my case, start a Vue 3 project with Vite and follow the Uppy Vue docs: https://uppy.io/docs/vue/

The example code triggers a typescript error:
```
frontend/components/UploadZone.vue:14:20 - error TS2554: Expected 0 arguments, but got 1.

14   uppy.value.close({ reason: "unmount" });
                      ~~~~~~~~~~~~~~~~~~~~~
```

```
<script setup lang="ts">
import { Dashboard } from "@uppy/vue";

import "@uppy/core/dist/style.css";
import "@uppy/dashboard/dist/style.css";

import Uppy from "@uppy/core";
import Webcam from "@uppy/webcam";
import { computed, onUnmounted } from "vue";

const uppy = computed(() => new Uppy().use(Webcam));

onUnmounted(() => {
  uppy.value.close({ reason: "unmount" }); // <-- this line triggers the error
});
</script>

<template lang="pug">
.container
  Dashboard(:uppy="uppy" :plugins="['Webcam']")
</template>

<style scoped lang="scss"></style>
```

Which IS what the type definition says: https://github.com/transloadit/uppy/blob/c59e61c/packages/%40uppy/core/types/index.d.ts#L374

The implementation of the close function definitely has an object with the reason as a parameter.

### Expected behavior

Typescript types should match the implementation.

### Actual behavior

Typescript compliation fails due to the above mentioned error.

--- a/packages/@uppy/core/types/index.d.ts
+++ b/packages/@uppy/core/types/index.d.ts
@@ -32,6 +32,8 @@ type LocaleStrings<TNames extends string> = {
 
 type LogLevel = 'info' | 'warning' | 'error'
 
+type CancelOptions = { reason: 'user' | 'unmount' }
+
 export type Store = UppyUtils.Store
 
 export type InternalMetadata = UppyUtils.InternalMetadata
@@ -352,7 +354,7 @@ export class Uppy {
     UploadResult<TMeta>
   >
 
-  cancelAll(): void
+  cancelAll(options: CancelOptions): void
 
   retryUpload<TMeta extends IndexedObject<any> = Record<string, unknown>>(
     fileID: string
@@ -371,7 +373,7 @@ export class Uppy {
 
   removePlugin(instance: UIPlugin | BasePlugin): void
 
-  close(): void
+  close(options: CancelOptions): void
 
   logout(): void
 

